massa-versioning/src/lib.rs

```diff
@@ -9,13 +9,22 @@
 //! MIPState -> Deployment state of a MIPInfo
 //! MIPStore -> A map of MIPInfo -> MipState
 //!
-//! # Note on MipInfo versions
+//! # Notes on MipInfo versions
 //!
-//! There is 2 different versions:
+//! There is 2 different 'versions':
 //! * version == Network version -> This is the network version to announce and thus to store in block header
-//! * component_version -> This is the version for the associated component and is used in VersioningFactory
+//! * component_version -> This is the version for the associated component and is used in VersioningFactory (e.g KeyPair, Address, VM)
 //!
-//! # Note on MipState
+//! # Notes on MipInfo timings and stats
+//!
+//! So in the execution module and after a block become final, we update the MipStore stats (in a blocking way) then continue one.
```

Comment:
Then continue one? I don't understand what that means

Reply:
removed as it is misleading